.storybook/preview.tsx

```diff
@@ -32,7 +32,9 @@ const preview: Preview = {
 	decorators: [
 		(Story) => (
 			<ThemeProvider theme={theme}>
-				<Story />
+				<ScopedCssBaseline>
```

Comment:
Just to make sure, I believe @drskullster mentioned he was going to add this in a standalone PR. Was using this agreed upon?